### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by spiffxp**
This is mostly to verify https://github.com/kubernetes/test-infra/pull/10835

### CODE REVIEW

**Comment by spiffxp** [comment]
Yay, label auto-applied thanks to the root OWNERS file

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # k8s.io
+
 Kubernetes files for various sites and infrastructure.
+
+Please see https://github.com/kubernetes/community/tree/master/wg-k8s-infra for more information
```
Comment (nikhita) [inline README.md]: tiny nit: let's use the git.k8s.io link here?

Comment (spiffxp) [reply README.md]: done

**Comment by spiffxp** [comment]
/approve
self approve

**Pull request merged**